README.md

```diff
@@ -1,11 +1,15 @@
-## What is oah-bes-vm 
+## What is oah-bes-vm?
 
 A substrate that brings up bundles of BeS & other tools to perform Red Team & Blue Team activity.
 
 ## Who are the end users?
 
 RT & BT activist of BeS community.
 
+### Usage scenarios:
+1. **Security Analyst**: RT & BT activist of BeS community can use oah-bes-vm to work on a specific open-source security project.
```

Comment:
Link to explanation of who a Security Analyst is in the context of this scenario

Reply:
Added keywords.md file inside docs, will link it with the Security Analyst keyword after the current PR is merged.